mysite/missions/templates/missions/diffpatch/hints.html

```diff
@@ -24,52 +24,61 @@
     <h3>Quick reference</h3>
   </div>
     <div class="submodule-body body">
-    
-  <h3><b> Working with patch and diff</b></h3>
-  <p>When working with patch, you are dealing with a patchfile containing a difference listing produced by the <em>diff</em> program and applies those differences to               one or more original files, producing patched versions.</p>
-  
-  <p>When working with diff, it outputs the difference between two files, by comparing both files line by line.</p>
+
+  <h3><b>Working with patch and diff</b></h3>
+  <p>When working with patch, you are dealing with a patch file, which contains a
+  list of differences produced by the <em>diff</em> program. The patch file applies
+  those differences to one or more original files, producing patched versions.</p>
+
+  <p>In contrast, diff outputs the difference between two files by comparing both
+  files line by line.</p>
+
   <p>This page is just a quick reference; see the bottom for other resources.</p>
   <br>
-  
-  <p> To work with patch, you can use:<br><br>
- 	<b>patch [options] [originalfile [patchfile]]</b> <br><br> but the more common method is :<br><br>
  <b>patch -pnum < patchfile.patch</p></b>
-  
-  <p> -pnum is the option to tell it how many levels of directories to ignore so it finds the correct files to patch.</p>
-  
-  <p> To work with diff, you can use:<br><br>
- 	<b>diff -u original <em>updatedfile</em></b><br></p>
- 	<p>For window users, redirect the output of your diff to a new file with this command:<br><br>
-	<b>diff -u original new > output.txt</b></p>
-	
-	<p> <b>-u</b> is the Output NUM (default 3) lines of unified context. </p>
-	<p> <b>-ur</b> is used when comparing directories of unified context by recursively comparing any subdirectories found.</p>

-  
-  
+
+  <p>To work with patch, you can use:</p><br>
+  <b>patch [options] [originalfile [patchfile]]</b><br>
+  <p>but the more common method is:<br><br></p>
+  <b>patch -pnum &lt; patchfile.patch</b>
+
+  <p>-pnum is the option to tell patch how many levels of directories to ignore so
+  it finds the correct files to patch.</p>
+
+  <p>To work with diff, you can use:<br><br>
+  <b>diff -u original <em>updatedfile</em></b><br></p>
+
+  <p>For Windows users, you can redirect the output of your diff to a new file
+  with this command:<br><br>
+  <b>diff -u original new > output.txt</b></p>
+
+  <p><b>-u</b> is the Output NUM (default 3) lines of unified context.</p>
+  <p><b>-ur</b> is used when comparing directories of unified context by recursively comparing any subdirectories found.</p>
```

Comment:
OH, I see now. I think there were a lot of intermingled tabs and spaces that I was trying to resolve. Will work on that now!